src/styles/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: 50%;
+  margin-right: 0;
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+  margin-right: 4px;
+}
```

Comment:
reuse stars from previous block and keep it operational so we can modify number of active stars

Reply:
not fixed